File: packet/entity.ts
export interface Entity {
    id: number;
}

export interface UserStatus extends Entity {
    name: string;
    level: number;
    lastLogin: Date;
    create: Date;
}

export interface Item extends Entity {
    name: string;
    level: number;
    grade: number;
    breakable: boolean;
}

File: packet/entity.cs
using System;

namespace Common
{
    public abstract class Entity
    {
         public int id;
    }

    public class UserStatus : Entity
    {
        public string name;
        public int level;
        public DateTime lastLogin;
        public DateTime create;
    }


    public class Item : Entity
    {
        public string name;
        public int level;
        public int grade;
        public bool breakable;
    }

}



